### ISSUE TITLE
Pivot does not have an accessible name 

### ISSUE THREAD

**Issue by taparikh** [OfficeDev/office-ui-fabric-react#11072]
### Environment Information

- **Package version(s)**: 7.8.2
- **Browser and OS versions**: Chrome Version 77.0.3865.120 (Official Build) (64-bit)

> Note that our screen reader support is limited to Edge + Narrator. Issues that do not reproduce in Edge + Narrator, and aren't caused by obvious invalid aria values, should be filed with the respective screen reading software, not the Fabric repo.

### Describe the issue:
Pivot does not have an accessible name when tested using the Accessibility Insights tool under the Custom Wdgets > Design Pattern Test (Assessment) :: We see <no value> 
When I check in the inspect element I do see the aria-label .

### Please provide a reproduction of the issue in a codepen:
https://codepen.io/pen?template=rEjZeV

#### Actual behavior:
The accessible name does not appear. 

#### Expected behavior:
The accessible name should appear

### Documentation describing expected behavior
This information is from the Accessibility Insights tool : A widget is an interactive interface component, such as a link, button, or combo box.  A custom widget is an interactive interface component other than a link or native HTML element. Custom widgets can be simple (e.g., a link or a button) or complex (e.g., a text field, listbox, and button that together function as a combo box). Each custom widget should follow the ARIA design pattern that best describes its function.

Below is a screenshot for the issue. 
![acc2](https://user-images.githubusercontent.com/51679982/68171154-7c8e0380-ff27-11e9-84e5-c6e8284746c1.JPG)

Below is a screenshot showing that aria-label gets applied to the component but still does not show up in the Accessibility insights test. 
![acc_1](https://user-images.githubusercontent.com/51679982/68171179-9c252c00-ff27-11e9-8c97-9de72a2930d5.JPG)

**Comment by JasonGore** [comment]
The CodePen you linked seems to be a template.

However, I was still able to take a look at this and confirm what you are seeing. Even when the `Pivot` element containing the `role` of `tablist` has an `aria-label`, Accessibility Insights still shows `Accessible name: (no value)`. This means there isn't a fix I can identify from Fabric's side (at least not yet.) I suggest filing an issue with Accessibility Insights repos (repo links are in the [wiki](https://github.com/OfficeDev/office-ui-fabric-react/wiki/Accessibility-Troubleshooting#troubleshooting).)

![image](https://user-images.githubusercontent.com/26070760/68230677-59f2fd80-ffae-11e9-8e6a-797d11486b18.png)

**Comment by taparikh** [comment]
Due to the heavy data dependency, I was unable to isolate the component and make it runnable on the codepen
Happy to hop on a quick call and showcase the issue to you for a deeper dive.

**Comment by taparikh** [comment]
@JasonGore   Can you take a look at our detailsList & Scrollpane snippet below and let us know if we are missing anything ? 

`<div
          style={{
            height: "calc(100% - 60px)",
            width: "calc(100% - 300px)",
            position: "relative",
            overflow: "hidden",
            marginLeft: "20px"
          }}
        >
          <ScrollablePane aria-label="scrollable pane label" id="scroll-pane">
            <DetailsList
              ariaLabelForGrid={this.props.gridAriaLabel}
              aria-description={this.props.gridAriaDesc}
              setKey="items"
              items={this.state.items}
              selection={this._selection}
              columns={columns}
              constrainMode={ConstrainMode.unconstrained}
              checkboxVisibility={checkboxVisibility}
              isHeaderVisible={isHeaderVisible}
              selectionMode={selectionMode}
              layoutMode={DetailsListLayoutMode.fixedColumns}
              enterModalSelectionOnTouch={true}
              checkButtonAriaLabel={"Central Grid check box"}
              onItemInvoked={this._onItemInvoked}
              selectionZoneProps={{
                selection: this._selection,
                disableAutoSelectOnInputElements: true,
                selectionMode: selectionMode
              }}
              ariaLabelForListHeader="Column headers."
              ariaLabelForSelectAllCheckbox="Toggle selection for all items"
              ariaLabelForSelectionColumn="Toggle selection"
              onRenderDetailsHeader={(
                detailHeaderProps: IDetailsHeaderProps,
                defaultRender: IRenderFunction<IDetailsHeaderProps>
              ) => (
                <Sticky stickyPosition={StickyPositionType.Header}>
                  {defaultRender(detailHeaderProps)}
                </Sticky>
              )}
            />
          </ScrollablePane>
        </div>`

**Comment by JasonGore** [comment]
I'm pretty sure I duplicated what you found in the image I attached to the post above. It is still not identifying it by name, so at the very least, we'd need their guidance anyways (if it is in fact not an AI issue.)

**Comment by qazaahir** [comment]
To add more context I see the same issue with other elements as well
![image](https://user-images.githubusercontent.com/57369169/68342263-14622d80-009f-11ea-9676-756997b442e9.png)

![image](https://user-images.githubusercontent.com/57369169/68342322-322f9280-009f-11ea-9483-80978e35437e.png)

**Comment by JasonGore** [comment]
Accessible Name is not directly provided by the code, it is a [computed value](https://www.w3.org/TR/accname-1.1/) that browsers (and in this case, Accessibility Insights plugin) calculate. Per my previous posts, I think this is an issue that needs to be brought up to Accessibility Insights first. If there is something the Fabric page is doing incorrectly, it's not obvious to me and not being brought up by Accessibility Insights as a failure.

**Comment by taparikh** [comment]
The aria-label does not get added at that level. Below is a screenshot that shows this. 
![Screenshot of code without aria-label](https://user-images.githubusercontent.com/51679982/70674361-9d3e2d00-1c3a-11ea-8064-b3ca82ceabf6.png) 
Notice that there is not aria-label for the role="tablist". 

![screenshot of Accessibility properties for tablist component](https://user-images.githubusercontent.com/51679982/70674473-ec845d80-1c3a-11ea-9041-d7adcd50ede0.png)
Notice that the browser is also not computing accessible name for tablist. 

`<div data-is-scrollable={false}>
        <ErrorBoundary>
          <Pivot
            role="navigation"
            aria-label="Top Navigation Bar"
            className={classNames.pivot}
            selectedKey={this.state.choice}
            onLinkClick={this.onChoiceChange}
          >
            <PivotItem
              role="navigation"
              headerText="Find Content"
              itemIcon="Search"
              itemKey={"find"}
              ariaLabel={"FindContent"}
            >`

This is the code snippet using the pivot component with the aria-label prop.

**Comment by taparikh** [comment]
Here is the thread of discussion with the Accessibility Insights Team : https://github.com/microsoft/accessibility-insights-web/issues/1639

**Comment by taparikh** [comment]
@JasonGore We would like to drive resolution to this bug. Can we hop on a teams call ?

**Comment by taparikh** [comment]
Tested this out on a screen reader and it was not reading the accessible name of the pivot. It would directly read the first tab name  …

**Comment by JasonGore** [comment]
I don't remember how I originally had applied `aria-label` to `Pivot` in my previous comment but it does not seem possible to do via props. I think the fix here is to apply props either to the inner `div` where `role` is defind or move the `role` up to the outer `div`.

**Comment by JasonGore** [comment]
Reopening per #11690

**Comment by xugao** [comment]
I believe this specific issue can be considered fixed in #11690. not the pivot root now has role `toolbar` instead of `tablist`, and it contains parts `tablist` and `tabpanel`. 
@qazaahir - feel free to verify
